src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -5707,6 +5713,12 @@ public void CopyTo(System.Span<char> destination) { }
         public int IndexOf(string value, int startIndex, int count, System.StringComparison comparisonType) { throw null; }
         public int IndexOf(string value, int startIndex, System.StringComparison comparisonType) { throw null; }
         public int IndexOf(string value, System.StringComparison comparisonType) { throw null; }
+        public int IndexOf(System.Text.Rune value) { throw null; }
+        public int IndexOf(System.Text.Rune value, int startIndex) { throw null; }
+        public int IndexOf(System.Text.Rune value, int startIndex, int count) { throw null; }
+        public int IndexOf(System.Text.Rune value, System.StringComparison comparisonType) { throw null; }
+        public int IndexOf(System.Text.Rune value, int startIndex, System.StringComparison comparisonType) { throw null; }
+        public int IndexOf(System.Text.Rune value, int startIndex, int count, System.StringComparison comparisonType) { throw null; }
         public int IndexOfAny(char[] anyOf) { throw null; }
```

Comment:
I am seeing the following are not of the proposal either, same if this is not design approved we need to track those first for review before implementing it.

        public int IndexOf(System.Text.Rune value, int startIndex, System.StringComparison comparisonType) { throw null; }
        public int IndexOf(System.Text.Rune value, int startIndex, int count, System.StringComparison comparisonType) { throw null; }

Reply:
Yes, please go ahead and remove them. Thanks!

Why do we need to mark them as private? Any unused code should be deleted (except if it is used as a helper method).

Reply:
I am fine either way.